src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -497,4 +497,7 @@
         Items="@(CorrelationFile)"
         Overwrite="true" />
   </Target>
+
+  <!-- Setup Dumpling service to collect and upload any crash dumps -->
+  <Import Project="$(MSBuildThisFileDirectory)Dumpling.targets" />
```

Comment:
Ah yes, I didn't catch that. I agree that this should only be enabled with a property switch.